### ISSUE TITLE
Tables should fill the available window width if possible instead of having a scrollbar

### ISSUE THREAD

**Issue by ianhinder** [UoMResearchIT/CapX#158]
The tasks table, with a wide browser window, keeps a fixed size and even has its own scrollbar, when there is plenty of horizontal space in the browser. It's annoying to have to scroll in this situation.

![image](https://github.com/UoMResearchIT/CapX/assets/213274/19f101b4-2b0a-4a9e-9f84-13644bea8bc7)

**Comment by aharwood2** [comment]
Ah, gotta love CSS. This is using some kind of default Blazor styling so that it is supposed to resize down when the screen shrinks to "mobile friendly". @PhilBradbury never liked this either. Feel free to find the needle in a haystack property combination that fixes this without breaking the shrinking!

**Comment by aharwood2** [comment]
So I got to the bottom of this. The `standard.css` file that is used by Radzen.Blazor uses a series of `@media` styles to detect screen resolution and adjust the `max-width` property of the `container` element. However, these are done in seemingly arbitrary steps and stop at a detected width of 1200px:

![Image](https://github.com/UoMResearchIT/CapX/assets/8614261/d2db3031-35f9-49ae-8bde-01f1b94bac5b)

Which means that for any device that has a window width >1200px the css file automatically limits the content width to 1140px. I've adjusted these in the `site.css` which overrides the `standard.css` by adding a few more increments (via trial and error) and then stating that for windows >2560px it just turns off the `max-width` property. Not sure how optimal this is for people but I used steps of what appears to fairly standard "screen resolutions" so hopefully a better use of screen real estate and less scrolling:

![Image](https://github.com/UoMResearchIT/CapX/assets/8614261/870361a8-790b-438a-97e9-9244071bc5ef)
